test/test_helper.cpp

```diff
@@ -1,18 +1,17 @@
-#include "unit_test_helper.h"
```

Comment:
The initial idea was to just move some functions into a specific file. Now this file has an extra set of functions which are not needed for performance tests. Could you please put back functions which are not common in `unit_test_helper.cpp` back? I'm sorry for such extra move